src/components/icon-as-button/IconAsButton.jsx

```diff
@@ -1,22 +1,52 @@
+// @flow
 import React from 'react';
-import PropTypes from 'prop-types';
+import type {Node} from 'react';
 import classNames from 'classnames';
-import Icon, {TYPE, ICON_COLOR} from '../icons/Icon';
+import * as IconModule from '../icons/Icon';
```

Comment:
`import as *` is only way to import types in a way compatible with our vs code intellisence

Reply:
Why not just as Icon ?

Reply:
ah to spread it 👍